lib/config.ml

```diff
@@ -65,8 +68,17 @@ let auto_release =
     ~docv:"SEC"
     ["confirm-auto-release"]
 
+let state_dir =
+  Arg.value @@
+  Arg.(opt string) (Filename.concat (Sys.getcwd ()) "var") @@
+  Arg.info
+    ~doc:"Set the state directory for the current pipeline."
+    ~docv:"DIR"
+    ["current-state-dir"]
```

Comment:
```suggestion
    ~doc:"Set the state directory for the pipeline."
    ~docv:"DIR"
    ["state-dir"]
```
(or use `ocurrent` instead of `current`, but none of the other options are prefixed this way)